css/properties/accent-color.json

```diff
@@ -75,6 +73,46 @@
               "deprecated": false
             }
           }
+        },
+        "maintains_contrast": {
+          "__compat": {
+            "description": "Maintains contrast",
+            "spec_url": "https://drafts.csswg.org/css-ui/#:~:text=must%20maintain%20contrast%20for%20legibility%20of%20the%20control",
+            "tags": [
+              "web-features:accent-color"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": false,
```

Comment:
I think this is slightly erroneous. I believe it works on desktop (though obviously will need to be confirmed) just not on Android.

Reply:
Good catch, thanks, you're right, Chrome maintains contrast since version 93, just Chrome Android doesn't. I will open a follow-up PR.